lib/sql.js

```diff
@@ -1159,8 +1159,9 @@ SQLConnector.prototype._buildWhere = function(model, where) {
  * @returns {string} The ORDER BY clause
  */
 SQLConnector.prototype.buildOrderBy = function(model, order) {
+  var idName = this.idName(model);
```

Comment:
I'm a bit concerned with this change as it feels like we are masking over a real issue by forcing an order on the dataset.  Has anyone tested outside of Node.js or Loopback whether or not Oracle returns the dataset in differing orders on subsequent calls to a SQL statement that does not include an ORDER BY clause?

@raymondfeng thoughts on this?